templates/accounts/home.html

```diff
@@ -1,6 +1,6 @@
-{% extends 'accounts/base.html' %}
+{% extends 'base/base.html' %}
 
-{ % block tiitle % }ホーム{% endblock title}
+{ % block tiitle % }ホーム{% endblock title % }
```

Comment:
```suggestion
{% block title %}ホーム{% endblock title %}
```
`{ %`のように { と % の間にスペース入れるとテストエラーになりませんでしたか？こちら出ていなくてもエラー原因に変わりないので、スペース入れないでください。
また`{% endblock title %}`にするならば、他のところも`{% endblock title %}`で統一してください。`{% endblock %}`に統一するなら、`{% endblock %}`に統一するようにお願いします。